app/scripts/directives/questionToolbar.js

```diff
@@ -170,16 +170,18 @@ angular
           },
           {
             id: 'campusQuestion',
-            defaultTitle: 'Campus',
+            defaultTitle:
+              'Campus- Type the full name of your campus and choose from the list that appears',
```

Comment:
I believe we need a space between `Campus` and the dash.
```suggestion
              'Campus - Type the full name of your campus and choose from the list that appears',
```